### ISSUE TITLE
kivy/lib/mtdev.py: ctypes "POINTER" should be "pointer"

### ISSUE THREAD

**Issue by pietervandermeer** [kivy/kivy#5456]
Line 155 in kivy/lib/mtdev.py
```
        mtdev_close(POINTER(self._device))
```
should be 
```
        mtdev_close(pointer(self._device))
```
the former creates a pointer type, the latter creates an actual pointer instance. this solves a crash.

**Comment by KeyWeeUsr** [comment]
Thanks for reporting the issue. If you can, please open a pull request with the change, so that you get the credits for it. Also, if you can, please provide the crash log to this issue. :)

**Comment by pietervandermeer** [comment]
np. i'm in no need of credits. there has been a proposed fix for this by someone else, coupled with an actual commit, but i don't know why it never made it to the trunk?

**Comment by KeyWeeUsr** [comment]
Aaaah, I think it's #5206. I'm not sure either, but from the discussion I believe even the CPU issues were figured out. Let's merge it to master and wait if something weird pops out. Doesn't look like it though.

**Comment by KeyWeeUsr** [comment]
Closed by #5206
